runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkRunner.java

```diff
@@ -81,6 +82,11 @@ public PipelineResult run(Pipeline pipeline) {
       SplittableParDo.convertReadBasedSplittableDoFnsToPrimitiveReadsIfNecessary(pipeline);
     }
 
+    if (!options.isStreaming()
```

Comment:
What is the risk for Streaming pipelines?

Reply:
@angoenka mentioned that updates for streaming pipelines might fail if we change the pipeline graph.